lms/djangoapps/instructor/views/api.py

```diff
@@ -2821,21 +2821,35 @@ def start_certificate_regeneration(request, course_id):
     """
     course_key = CourseKey.from_string(course_id)
     certificates_statuses = request.POST.getlist('certificate_statuses', [])
-    if not certificates_statuses:
+    verified_with_audit_certs = request.POST.get('verified_but_audit_certs')
```

Comment:
keep the naming consistent—either `'verified_but_audit_certs'` or `'verified_with_audit_certs'`